examples/src/charmory_examples/object_detection/xview/train_xview_model.py

```diff
@@ -0,0 +1,342 @@
+    61: 41,  # Haul Truck
+    62: 42,  # Scraper/Tractor
+    63: 43,  # Front Loader/Bulldozer
+    64: 44,  # Excavator
+    65: 45,  # Cement Mixer
+    66: 46,  # Ground Grader
+    71: 47,  # Hut/Tent
+    72: 48,  # Shed
+    73: 49,  # Building
+    74: 50,  # Aircraft Hanger
+    75: 51,  # Unknown1
+    76: 52,  # Damaged Building
+    77: 53,  # Facility
+    79: 54,  # Construction Site
+    82: 55,  # Unknown2
+    83: 56,  # Vehicle Lot
+    84: 57,  # Helipad
+    86: 58,  # Storage Tank
+    89: 59,  # Shipping Container Lot
+    91: 60,  # Shipping Container
+    93: 61,  # Pylon
+    94: 62,  # Tower
+}
+
+def load_huggingface_dataset():
+    s3 = S3FileSystem(anon=False)
+    train_dataset = load_from_disk("s3://armory-library-data/datasets/train/", fs=s3)
+    new_dataset = train_dataset.train_test_split(test_size=0.3, seed=1)
+    train_dataset, test_dataset = new_dataset["train"], new_dataset["test"]
+
+    return train_dataset, test_dataset
+train_dataset, test_dataset = load_huggingface_dataset()
+
+class Averager:
+    def __init__(self):
+        self.current_total = 0.0
+        self.iterations = 0.0
+        
+    def send(self, value):
+        self.current_total += value
+        self.iterations += 1
+    
+    @property
+    def value(self):
+        if self.iterations == 0:
+            return 0
+        else:
+            return 1.0 * self.current_total / self.iterations
+    
+    def reset(self):
+        self.current_total = 0.0
+        self.iterations = 0.0
+from albumentations.pytorch import ToTensorV2
+def collate_fn(batch):
+    """
+    To handle the data loading as different images may have different number 
+    of objects and to handle varying size tensors as well.
+    """
+    return tuple(zip(*batch))
+def get_train_transform():
+    return A.Compose([
+        A.Flip(0.5),
+        A.RandomRotate90(0.5),
+        A.MotionBlur(p=0.2),
+        A.MedianBlur(blur_limit=3, p=0.1),
+        A.Blur(blur_limit=3, p=0.1),
+        ToTensorV2(p=1.0),
+    ], bbox_params={
+        'format': 'pascal_voc',
+        'label_fields': ['labels']
+    })
+# define the validation transforms
+def get_valid_transform():
+    return A.Compose([
+        ToTensorV2(p=1.0),
+    ], bbox_params={
+        'format': 'pascal_voc', 
+        'label_fields': ['labels']
+    })
+
+import cv2
+from torch.nn import CrossEntropyLoss
+from torch.nn import MSELoss
+classLossFunc = CrossEntropyLoss()
+bboxLossFunc = MSELoss()
+
+class xview(torch.utils.data.Dataset):
+    def __init__(self,dataset, width, height,  transforms=None):
+        self.transforms = transforms
+        self.image_dataset = dataset
+        self.height = height
+        self.width = width
+  
+
+    def __getitem__(self, idx):
+        img = self.image_dataset[idx]
+
+
+        image = np.asarray(img['image'])
+        image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB).astype(np.float32)
+        image_resized = cv2.resize(image, (self.width, self.height))
```

Comment:
Could the resize have been done using albumentations?

Reply:
Yes that should be fixed now